packages/main/src/themes/Toast.css

```diff
@@ -0,0 +1,89 @@
+:host {
+	font-family: var(--sapUiFontFamily);
```

Comment:
And as we discussed the "sapUi"  params would be removed and should be replaced by the "sap" ones:

sapuiFontFamily            -> sapFontFamily (available)
sapUiListTextColor        -> sapTextColor (available)
sapMFontMediumSize  -> sapFontMediumSize (available soon)
sapUiListBackground    -> sapList_Background (available)
sapUiShadowLevel2      -> sapContent_Shadow2 (available soon)